bin/rocm-test/scripts/parse_LLNL.sh

```diff
@@ -49,6 +52,10 @@ function parse(){
       fi
       [[ "$line" =~ $testname_regex ]]
       llnltest=${BASH_REMATCH[1]}
+      if [ "$outfile" = "failing-tests.txt" ] && \
+	 grep -Fq "$llnltest" xfail-list then
```

Comment:
Missing a semi-colon here after `xfail-list`.

Reply:
...huh, but yeah. I wonder why it didn't complain! I'll fix that, thanks.